Xamarin.Essentials/Barometer/Barometer.android.cs

```diff
@@ -0,0 +1,43 @@
+﻿using System;
+using Android.Hardware;
+using Android.Runtime;
+
+namespace Xamarin.Essentials
+{
+    public static partial class Barometer
+    {
+        internal static bool PlatformIsSupported =>
+               barometer.Value != null;
+
+        static readonly Lazy<Sensor> barometer = new Lazy<Sensor>(
```

Comment:
No need to use a lazy here, can be removed to follow the pattern of others as they need to be reset.

Reply:
Same here, I don't think it needs to be reset. The android compass also does not null or dispose the sensor, merely reset it. As a barometer is not really something I would consider pluggable and portable to change support of during runtime I think we can actually use a lazy and get away better than if we were to use the current implementation.